addons/html_editor/static/src/main/table/table_drag_drop.js

```diff
@@ -0,0 +1,172 @@
+import { closestElement } from "@html_editor/utils/dom_traversal";
+import { getColumnIndex, getRowIndex } from "@html_editor/utils/table";
+import { Component, useRef, useExternalListener, onMounted, onWillUnmount } from "@odoo/owl";
+
+const OVERLAY_CLAMP_OFFSET = 5;
+
+export class TableDragDrop extends Component {
+    static template = "html_editor.TableDragDrop";
```

Comment:
I don't understand why this feature needs a template? I couldn't find the template in the DOM when inspecting while using the feature.